dev/com.ibm.ws.io.leangen.graphql.spqr.0.9.9/bnd.bnd

```diff
@@ -0,0 +1,36 @@
+#*******************************************************************************
```

Comment:
Should we leave the version number off of this project name? ```com.ibm.ws.io.leangen.graphql.spqr``` seems more appropriate, especially since we don't have multiple versions of ```leangen.graphql.spqr``` in Liberty that we need to differentiate between.

Reply:
That being said it's probably best to be consistent across all projects... As this gets updated would the project be renamed to 1.0, 1.1, 2.0 etc? I do think it should be ```com.ibm.ws.io.leangen.graphql.spqr.0.9``` at the very least, we normally stop and major and minor number.

Reply:
Per conversation in person, we're planning to delete the 0.9.9 project and replace it with a project that pulls in SPQR (with the MP-specific code) directly from Maven Central once it is available.